hamlpy/parser/filters.py

```diff
@@ -1,7 +1,13 @@
 from __future__ import unicode_literals
+"""
+Core HamlPy filters.
+
+The implementation of these should match https://github.com/haml/haml/blob/master/lib/haml/filters.rb as closely as
+possible. Where we differ is that we don't compile Stylus, Coffeescript etc into CSS or Javascript - but place the
+content into suitable <style> and <script> that can be transformed later by something like django-compressor.
+"""
```

Comment:
I feel like I only just about understand how a `:coffee` section currently ends up as compressed js, but wouldn't that mean that the coffee&rarr;js compilation has to happen for every request? Is it time we tried using cached templates?